dataverse-webapp/src/main/resources/config/dataverse.default.properties

```diff
@@ -45,7 +45,7 @@ HomePageCustomizationFile=
 InheritParentRoleAssignments=
 LogoCustomizationFile=
 MetricsUrl=
-ProvCollectionEnabled=false
+ProvCollectionEnabled=true
```

Comment:
Should this be enabled by default?

Reply:
IMO while on non production environment we shouldn't hide functionalities from developers/testers.
I'd keep this enabled.

Reply:
But default values are not made for developer env, otherwise we would have DoiProvider=FAKE. At least that was my assumption, IMO this is to be discussed in team since we are changing the approach.

Reply:
Adding `DoiProvider=FAKE`